cli/testing/echo.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2025 The Jujutsu Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::io::Write as _;
+
+use anyhow::Context as _;
+use anyhow::Result;
+
+fn main() -> Result<()> {
```

Comment:
nit: just unwrap the error instead of adding new dependency just for `echo`.

Reply:
Done. Use `unwrap_or_else` and `panic` to provide more information in case the program fails.